controlplane/api/v1alpha2/groupversion_info.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 // Package v1beta1 contains API Schema definitions for the controlplane v1beta1 API group
```

Comment:
Oops looks like we missed this comment the first time 
```suggestion
// Package v1alpha2 contains API Schema definitions for the controlplane v1alpha2 API group
```